cmd/helm/root.go

```diff
@@ -252,7 +252,7 @@ func checkForExpiredRepos(repofile string) {
 		if url := r.URL; strings.Contains(url, exp.old) {
 			fmt.Fprintf(
 				os.Stderr,
-				"WARNING: %q is deprecated for %q and will be deleted Nov. 13, 2020.\nWARNING: You should switch to %q\n",
+				"WARNING: %q is deprecated for %q and will be deleted Nov. 13, 2020.\nWARNING: You should switch to %q\nWARNING: Switch via: helm repo add stable https://charts.helm.sh/stable --force-update",
```

Comment:
So this message would potentially also be used for `incubator`. It might be best then to update it as follows:

helm repo add stable https://charts.helm.sh/stable --force-update --> helm repo add %q %q --force-update

Then add the addition params to the `Fprintf` function of:
exp.name, exp.new